bin/deku_node.re

```diff
@@ -208,16 +208,17 @@ let node = (folder, trusted_validator_membership_change_file) => {
       current_validators
       |> List.mapi((i, validator) => {
            (
-             validator,
+             Address.of_wallet(validator),
```

Comment:
That's wrong, you're fetching the validators from the storage, it will be an address already

Reply:
Fixed because this PR is fixes some problems